waku/api.go

```diff
@@ -227,6 +228,7 @@ type NewMessage struct {
 // Post posts a message on the Waku network.
 // returns the hash of the message in case of success.
 func (api *PublicWakuAPI) Post(ctx context.Context, req NewMessage) (hexutil.Bytes, error) {
+	logutils.ZapLogger().Info("AK Post", zap.String("stack", string(debug.Stack())))
```

Comment:
this might log a lot, consider removing or changing the log level to trace (or something like that)